std/math.d

```diff
@@ -4646,13 +4613,20 @@ real nearbyint(real x) @safe pure nothrow @nogc
  * $(LREF nearbyint) performs the same operation, but does
  * not set the FE_INEXACT exception.
  */
-real rint(real x) @safe pure nothrow @nogc { pragma(inline, true); return core.math.rint(x); }
-//FIXME
+real rint(real x) @safe pure nothrow @nogc
+{
+    pragma(inline, true); return core.math.rint(x);
```

Comment:
Please keep in mind that these `pragma(inline, true)` *statements* inside non-templates are ignored by LDC when imported by another module (function body needs semantic...).

Reply:
Ahh, noted. Although I suspect that LLVM's inline will be smart enough to do that with the hint.

Reply:
That hint really needs to be outside the function, otherwise there's no LLVM hint ;) - our std.math is full of additions like this: https://github.com/ldc-developers/phobos/blob/0d04cf72770a356033e8e5fca657b9b7316bc957/std/math.d#L1614